.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release new version
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        description: Type of release based om semver (e.g. patch, minor, major)
+        required: true
+        default: minor
+
+jobs:
+  release:
+    name: Release new version to NPM
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Setup Git
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+      - run: npm version ${{github.event.inputs.release-type}}
+      - name: Push commit and tag
+        run: |
+          git push
+          git push --follow-tags
```

Comment:
This doesn't work if there is any branch protection set up 😞 for `master`. GitHub will block this push.